devtools/cli/src/main/java/io/quarkus/cli/build/GradleRunner.java

```diff
@@ -136,22 +136,11 @@ public Integer projectInfo(boolean perModule) {
     }
 
     @Override
-    public Integer updateProject(String targetPlatformVersion, String targetPlatformStreamId, boolean perModule)
+    public Integer updateProject(String targetPlatformVersion, String targetPlatformStream, boolean perModule)
             throws Exception {
-        ArrayDeque<String> args = new ArrayDeque<>();
-        args.add("quarkusUpdate");
-        if (!StringUtil.isNullOrEmpty(targetPlatformVersion)) {
-            args.add("--platform-version");
-            args.add(targetPlatformVersion);
-        }
-        if (!StringUtil.isNullOrEmpty(targetPlatformStreamId)) {
-            args.add("--stream-id");
-            args.add(targetPlatformStreamId);
-        }
-        if (perModule) {
-            args.add("--per-module");
-        }
-        return run(prependExecutable(args));
+        MessageWriter.info().error(
+                "quarkus update is not yet available for Gradle projects. See https://github.com/quarkusio/quarkus/issues/31658 for more information.");
+        return 1;
```

Comment:
We should still call the Gradle task to keep what we have, this is still useful. Or am I missing something?

Reply:
Doesn't the current impl still display instructions to update to the currently recommended versions? I am not talking about applying OR recipes.